src/nvim/buffer.c

```diff
@@ -2605,14 +2605,22 @@ void buflist_list(exarg_T *eap)
       continue;
     }
 
+    const int changed_char = (buf->b_flags & BF_READERR)
```

Comment:
@justinmk You missed https://github.com/vim/vim/commit/304b64c9e6957fa3f552e0540ca786139b39a1c4#diff-a4fc02df73c5e2af124168e5b1436d00R3071 when working on https://github.com/neovim/neovim/pull/10370/.

Reply:
@janlazo Thanks, you mean `changed_char`? Will deal with it when I get around to https://github.com/neovim/neovim/pull/6280